test/stresses.jl

```diff
@@ -42,7 +43,7 @@ include("testcases.jl")
     stresses = compute_stresses(scfres)
     @test isapprox(stresses, compute_stresses(scfres_nosym), atol=1e-10)
 
-    dir = randn(3, 3)
+    dir = MPI.bcast(randn(3, 3), 0, MPI.COMM_WORLD)
```

Comment:
Nice catch! We should probably do this in PlaneWaveBasis, make sure all parameters (including Model) are the same on all processors...